MAINTAINERS

```diff
@@ -20,7 +20,7 @@ Jonas Östanbäck <jonas.ostanback@gmail.com> (@cez81)
 David Schneiderbauer <dschneiderbauer@gmail.com> (@daviian)
 Peter Žeby <morlinest@gmail.com> (@morlinest)
 Matti Ranta <matti@mdranta.net> (@techknowlogick)
-Jonas Franz <info@jonasfranz.software> (@JonasFranzDEV)
+Jonas Franz <info@jonasfranz.software> (@JonasFranz)
```

Comment:
New Username ist @jonasfranz

Reply:
I've updated PR to lowercase username to match